async_app.opam.locked

```diff
@@ -158,8 +158,6 @@ build: [
 ]
 dev-repo: "git+https://github.com/davesnx/async_app.git"
 pin-depends: [
-  [ "tyxml.dev" "git+https://github.com/jchavarri/tyxml.git#reason-react" ]
-  [ "tyxml-jsx.dev" "git+https://github.com/jchavarri/tyxml.git#reason-react" ]
-  [ "tyxml-syntax.dev" "git+https://github.com/jchavarri/tyxml.git#reason-react" ]
+  "tyxml.dev" "git+https://github.com/jchavarri/tyxml.git#reason-react"
```

Comment:
Was this intended? I think there is an issue currently were running lock will remove these packages, so they might have been committed after running `opam lock` 🤔